### ISSUE TITLE
[BUG] - Sound macro executes with no sound

### ISSUE THREAD

**Issue by Pentazer** [fantasycalendar/FoundryVTT-Sequencer#279]
**Describe the bug**
Sound sequence macro doesn't work?

Full path: `A:\Foundry\Data\_TTRPG_Master_Library\SFX\FFVI\SaveTingEcho.wav`
```
new Sequence()
    .sound()
        .file("_TTRPG_Master_Library/SFX/FFVI/SaveTing.wav")
        .fadeInAudio(0)
        .fadeOutAudio(0)
    .play()
```

According to Sequencer all I should need is this, as copied from the sequencer wiki:
```
new Sequence()
    .sound()
        .file("Music/Sound_Effects/Phoenix_Cry.wav")
        .fadeInAudio(500)
        .fadeOutAudio(500)
    .play()
```

I can't really figure out what I may be missing here, if I'm missing anything?

**To Reproduce**
Type up macro.
Click "execute macro"

**Expected behavior**
Plays the save ting wav file.

**Screenshots**
![image](https://github.com/user-attachments/assets/5a7e6513-8059-4fd6-8900-83430b99edb9)

**Setup:**
 - Foundry Version: 12.328
 - System Version: DND5e 3.3.1
 - Sequencer Version: 3.2.11
 - Browser & version: None, foundry desktop client.

**Active modules:**
 - socketlib
 - Module Management+
 - Sequencer
 - Automated Animations

**Additional context**
If I write the path improperly so as to ensure a failure to find the file, it does indeed fail and produces an error:
`foundry-esm.js:51491 Failed to load audio element "_TTRPG_aster_Library/SFX/FFVI/SaveTing.wav"`
Note the missing M in Master.

If the file is correctly sourced, no error occurs, but no sound plays.
If play the exact same sound using Automated animations it plays in foundry, which confirms foundry is playing audio, the file works and the directory functions.

**Comment by Lui-Sin** [comment]
This also happens in version 3.2.8 so it must be a bug introduced in that patch or earlier. In my case I'm using Foundry version 11.315 Stable.

**Comment by MrVauxs** [comment]
This likely has been fixed in the next version.
https://github.com/fantasycalendar/FoundryVTT-Sequencer/commit/a3edc36bc6a8756b9fe180f3ab4f9e7495444be5#diff-c1dda753dc3c6ae4b1e110376cf26240179cb9a045d9991b466ee08473204cabR6

**Comment by Haxxer** [comment]
Fixed in 3.2.12
